starter-code/index.js

```diff
@@ -1,17 +1,55 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.length = this.items.length;
+    this.items = this.items.sort((a, b) => a - b);
+  }
 
-  get(pos) {}
+  get(pos) {
+    let value = this.items[pos];
+    if (pos === this.items[pos]) {
+      return value;
+    } else {
+      throw new Error("OutOfBounds");
```

Comment:
This function is a little confusing. You could have checked directly for the pos value and from there do your returns.

It's totally correct, but a little confusing for my head, at least